frontend/package.json

```diff
@@ -96,14 +96,15 @@
     "react-dnd": "^2.6.0",
     "react-dnd-html5-backend": "^2.6.0",
     "react-dom": "16.8.6",
-    "react-helmet": "5.x",
+    "react-helmet": "5.2.1",
```

Comment:
Do we need to lock to specific versions or can this be `^5.2.1` etc?

Reply:
Whenever I update dependencies, I run `yarn add <package>@<most-recent-semver>`.

Reply:
I'd prefer to specify a range so that `yarn upgrade` works. We should be consistent across dependencies, and `^` is the norm currently.

Reply:
Done